zhmcclient/_adapter.py

```diff
@@ -347,3 +347,44 @@ def __repr__(self):
                 _ports=repr_manager(self._ports, indent=2),
             ))
         return ret
+
+    @logged_api_call
+    def change_crypto_type(self, crypto_type, zeroize=None):
+        """
+        Reconfigures a cryptographic adapter to a different crypto type.
+        This operation is only supported for cryptographic adapters.
+
+        Authorization requirements:
+
+        * Object-access permission to this Partition.
+        * Task permission to the "Partition Details" task.
+
+        Parameters:
+
+          crypto_type:
+            - 'accelerator':
+               Crypto Express4S Accelerator or
+               Crypto Express5S Accelerator.
+            - 'cca-coprocessor':
+               Crypto Express4S CCA Coprocessor or
+               Crypto Express5S Coprocessor.
+            - 'ep11-coprocessor':
+               Crypto Express4S EP11 Coprocessor or
+               Crypto Express5S EP11 Coprocessor.
```

Comment:
I have verified that the Crypto Express4S is not supported on z13. It should be removed from the list (and I'll open a defect against the HMC API book related to that).